src/token.coffee

```diff
@@ -144,6 +144,15 @@ class Token
       isHardTab: isHardTab
     )
 
+  buildPhantomTabToken: (length) ->
```

Comment:
How about `buildSoftWrapIndentToken`. I was confused at first thinking this had something to do with tab when it doesn't.